pandas/tests/test_categorical.py

```diff
@@ -1410,10 +1410,8 @@ def test_searchsorted(self):
         self.assert_numpy_array_equal(res, chk)
 
         # As above, but with a sorter array to reorder an unsorted array
-        res = c2.searchsorted(['bread', 'eggs'], side='right',
-                              sorter=[0, 1, 2, 3, 5, 4])
-        chk = s2.searchsorted(['bread', 'eggs'], side='right',
-                              sorter=[0, 1, 2, 3, 5, 4])
+        res = c2.searchsorted(['bread', 'eggs'], 'right', [0, 1, 2, 3, 5, 4])
```

Comment:
Well, I took your suggestion about the `try`-`except` to heart in my `numpy` PR so `numpy` won't ever be sending it in as `**kwargs` unless someone beats me to it.